pyproject.toml

```diff
@@ -17,8 +17,9 @@ readme = "README.md"
 dynamic = ["version"]
 
 dependencies = [
-    'aind-data-schema==0.31.8',
-    'pydantic<2.7'
+    'aind-data-schema==0.38.0',
+    'pydantic<2.7',
+    'tzlocal'
```

Comment:
If this is just being used in the tests, we should add it to the optional dependencies.

Reply:
cool